best-practices-trove-project-creation.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Creating projects for Trove"
+permalink: /best-practices-trove-project-creation/
+---
+# Creating a data collection project in Trove
+
+With Trove, you can create AI/ML data collection projects through trusted connections between you and the people who contribute to your projects, leading to higher quality data. This document explains how to create such a project in Trove.
+
+## Create Project in Trove Studio
+
+Start your data collection project by signing into the [Trove Studio](https://trove-studio.microsoft.com/) as a requester.  Once your account has been verified, click the "Create project" link to provide the details of your project.
+
+## Provide Clear and Concise Description and Details
+
+**Project Name** - keep this brief but concise ideally containing the object(s) and environment desired for example "Cigarettes on Beaches," "Clothing Stores" or "Pleasant Memories - Food."
+
+**Description** - provide a description in prose form highlighting the key objects in the desired context and environments to be captured in the photos you are looking for in as much detail as possible.  Also call out what the types of photos will _NOT_ be accepted.  Please note our requirement to indicate if the photos you desire will contain face data.  For example - "Send photos of a SPAM can in multiple places in your house (kitchen, living room, bedroom, bathroom). In each place, take at least 5 photos from different sides of the can (front, back, top, left, and right).  Do not send photos of a SPAM can on a shelf in a grocery store or photos containing people." 
+
+**How Photos will be used** - similar to the description, provide a summary in prose form of how these photos will be used and for what purpose.  For example - "These photos will be used to train artificial intelligence (AI) models to help improve the recognition of broken glass mugs and vessels in images for safety alerts."
```

Comment:
IMO, I would not say "similar to description". Reason because people shouldn't just copy+paste. Rather they should think about how to describe how the data will be used in a way that a normal person on the street understands. We could even say "if you had to explain to your grandpa how this data will be used should be how you write this.

Reply:
Great catch!  This section should focus on how the photos will be used.  My intent here was to call out following how the detailed clear description in the Description section was written, not merely asking them to cut and paste LOL.  I like that last sentence, adds a little color to the bland text - will include it.

Reply:
Done!